hhs_hosp/delphi_hhs/run.py

```diff
@@ -117,6 +117,12 @@ def run_module(params):
             df=pop_proportion(df, geo_mapper)
         df = make_geo(df, geo, geo_mapper)
         df = smooth_values(df, smoother[0])
+        # Fix N/A MA values, see issue #1360
+        if geo == "state" and sensor.startswith(CONFIRMED_FLU):
+            print(df.shape, geo, smoother, sensor)
+            df = df[~np.logical_and(np.logical_and(df.val.isna(), df.geo_id == "ma"),
+                                    df.timestamp > "08-01-2021")]
```

Comment:
Two suggestions on this bit:

- I'd prefer to make this filter even narrower by only removing `NA`s occurring on Tuesdays. Seems to me that we should be alerted to unusual missing values that occur on other days of the week.
- `&` can be used to element-wise combine arrays so that the chained logic here is clearer.